### PR TITLE
Compress remote dumps with gzip before downloading

### PR DESCRIPTION
**Pull request by triskweline**
Please review and release if OK.

### CODE REVIEW

File: lib/geordi/remote.rb [hunk 1]
```diff
@@ -34,18 +35,21 @@ def select_server
...
+      destination_path_gz = "#{destination_path}.gz"
+
       relative_destination = Pathname.new(destination_path).relative_path_from Pathname.new(@config.root)
+      relative_destination_gz = "#{relative_destination}.gz"
 
-      note "Downloading remote dump to #{relative_destination} ..."
+      note "Downloading remote dump to #{relative_destination_gz} ..."
       server = @config.primary_server
-      Util.system! "scp -C #{ @config.user(server) }@#{ server }:#{REMOTE_DUMP_PATH} #{destination_path}"
+      Util.system! "scp -C #{ @config.user(server) }@#{ server }:#{REMOTE_DUMP_PATH_GZ} #{destination_path_gz} && gunzip -f #{destination_path_gz}"
```
Comment (denzelem) [inline lib/geordi/remote.rb]: We already used the `-c` option within scp #52:

From the `scp` manual:

```
-C      Compression enable.  Passes the -C flag to ssh(1) to enable compression.
```

From the `ssh` manual:

```
-C      Requests compression of all data (including stdin, stdout, stderr, and data for
forwarded X11, TCP and UNIX-domain connections).  The compression algorithm is
the same used by gzip(1).  Compression is desirable on modem lines and other slow
connections, but will only slow down things on fast networks.  The default value can be
set on a host-by-host basis in the configuration files; see the Compression option.
```

@triskweline Do you think the manual compression before does provide any advantage?

**Comment by triskweline** [comment]
I didn't realize we were already compressing, no need to gzip twice.

I added another commit to this branch that helped me in development. Can you take a look if that's worth merging?
https://github.com/makandra/geordi/pull/85/commits/e9aba4901924c7fcdf8d69bbb6f8d8ba6496c11e

This allowed me to test this change from my local geordi repository folder:

    ~/Projects/geordi > bundle exec geordi dump stagin RAILS_ENV=~/Projects/other-project

**Comment by denzelem** [comment]
e9aba49 looks good to me. Thanks!

**Comment by triskweline** [comment]
I only merged that one commit then. Thanks for the review!

**Pull request closed without merging**